Makefile

```
@@ -0,0 +1,255 @@
+# Development Makefile for fmp-data
+# =====================================
+# Quick commands for common development tasks
+
+.PHONY: help install clean test lint format typecheck security docs
+.PHONY: test-all typecheck-all ci pre-commit fix check-all
+.PHONY: build publish-test publish smoke
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)